doc/optparse/argument_converters.rdoc

```diff
@@ -344,9 +344,17 @@ Executions:
 
 === Custom Argument Converters
 
+An argument converter has:
+
```

Comment:
Now the initial sentence "fights with" the following sentences, I think. Perhaps you can drop/reformulate, so that this sentence is linked to the bullet-point list.

Reply:
> 
> 
> Now the initial sentence "fights with" the following sentences, I think. Perhaps you can drop/reformulate, so that this sentence is linked to the bullet-point list.

I have deleted 347-8, which I think fixes the problem.

Reply:
👍 Shiny and clear! Thanks!